Lib/location.py

```diff
@@ -40,11 +40,11 @@ def stop_updating():
     __PyLocationHelper__.stopUpdating()
 
 
-def get_location() -> Tuple[float]:
+def get_location() -> Tuple[float, float, float]:
     """
     Returns a tuple with current longitude, latitude and altitude.
 
-    :rtype: Tuple[float]
```

Comment:
Why not return a namedtuple to be more self documenting?
```python
>>> from collections import namedtuple
>>> gps_location = namedtuple("gps_location", "longitude latitude altitude")
>>> gps_location(1, 2, 3)
gps_location(longitude=1, latitude=2, altitude=3)
```

Is the usual order long and then lat or the other way around??
https://en.wikipedia.org › wiki › Geographic_coordinate_system
I am used to lat, long, alt in that order.

Reply:
> Why not return a namedtuple to be more self documenting?

Definitely a good idea, although since Pyto is 3.8, I'd suggest going for `typing.NamedTuple` here:

```python
from typing import NamedTuple


class Location(NamedTuple):
    latitude: float
    longitude: float
    altitude: float
```

>  I am used to lat, long, alt in that order.

I definitely am too, but I didn't made that decision here :confused:
I think altitude is fine and commonly used.

(and even though `location` was introduced just recently, changing the order would be a breaking change.)

---

This PR is really just to fix the wrong type annotation :)

Reply:
Apple goes with lat first ans long second... https://developer.apple.com/documentation/corelocation/cllocation